pkg/indexmanagement/scripts.go

```diff
@@ -23,7 +23,7 @@ set -euo pipefail
 writeIndices=$(curl -s $ES_SERVICE/${POLICY_MAPPING}-*/_alias/${POLICY_MAPPING}-write \
   --cacert /etc/indexmanagement/keys/admin-ca \
   -H"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-  -HContent-Type:application/json)
+  -HContent-Type:application/json 2>&1)
```

Comment:
What is the goal with this redirect? I don't believe stderr is going to ever have anything anyway, because you need to add the `-S` flag to the curl command. In any case, if curl fails it will break the entire script and stderr would be printed out appropriately right?

Reply:
This is to ensure all outcome to the script is captured.